app/Models/V2/Projects/Project.php

```diff
@@ -359,17 +348,16 @@ public function getRegeneratedTreesCountAttribute(): int
 
     public function getWorkdayCountAttribute(): int
     {
-        return WorkdayDemographic::whereIn(
-            'workday_id',
-            Workday::where('workdayable_type', SiteReport::class)
-                ->whereIn('workdayable_id', $this->submittedSiteReports()->select('v2_site_reports.id'))
-                ->select('id')
-        )->orWhereIn(
-            'workday_id',
-            Workday::where('workdayable_type', ProjectReport::class)
-                ->whereIn('workdayable_id', $this->reports()->hasBeenSubmitted()->select('id'))
-                ->select('id')
-        )->gender()->sum('amount') ?? 0;
+        $sumQueries = [
+            DB::raw('sum(`workdays_paid`) as paid'),
+            DB::raw('sum(`workdays_volunteer`) as volunteer'),
+        ];
+        $projectTotals = $this->reports()->hasBeenSubmitted()->get($sumQueries)->first();
+        // The groupBy is superfluous, but required because Laravel adds "v2_sites.project_id as laravel_through_key" to
+        // the SQL select.
+        $siteTotals = $this->submittedSiteReports()->groupBy('v2_sites.project_id')->get($sumQueries)->first();
+
+        return $projectTotals?->paid + $projectTotals?->volunteer + $siteTotals?->paid + $siteTotals?->volunteer;
```

Comment:
This change is reverting some recent work on `staging`. Please update your branch from `staging` to avoid that.

Reply:
done merge from staging